gen/definitions/properties.yaml

```diff
@@ -627,3 +642,272 @@ fvIpAttr:
+    fwdCtrl: "The forwarding control of the %s object."
+    floodOnEncap: "Flood L2 Multicast/Broadcast and Link Local Layer based on encapsulation."
+    hasMcastSource: "The %s object has a multicast source."
+    isAttrBasedEPg: "The %s object is microsegmented (uSeg)."
+    pcEnfPref: "Parameter used to determine whether communication between endpoints within the EPG is blocked."
+    prefGrMemb: "Parameter used to determine whether the EPG is part of the preferred group. Members of this group are allowed to communicate without contracts."
+  test_values:
+    default:
+      forwarding_control: "none"
+    all:
+      forwarding_control: "proxy-arp"
+  exclude_targets:
+    - "fvESg"
+  parents:
+    - class_name: "fvAp"
+      parent_dependency: "fvTenant"
+
+fvRsAEPgMonPol:
+  documentation:
+    tnMonEPGPolName: "The name of the monitoring policy."
+
+fvRsFcPathAtt:
+  default_values:
+    vsan: "unknown"
+  test_values:
+    default:
+      vsan: "unknown"
+    all:
+      vsan: "vsan-10"
+    datasource_non_existing:
+      target_dn: "topology/pod-2/paths-201/pathep-[eth1/1]"
+  documentation:
+    vsan: "The virtual storage area network (VSAN) of the %s object."
+    vsanMode: "The virtual storage area network (VSAN) mode of the %s object."
+  parents:
+    - class_name: "fvAEPg"
+      parent_dependency: "fvAp"
+      parent_dn: "aci_application_epg.test.id"
+      target_classes:
+        - "fabricPathEp"
+  targets:
+    - class_name: "fabricPathEp"
+      target_dn: "topology/pod-1/paths-101/pathep-[eth1/1]"
+      relation_resource_name: "fibre_channel_path"
+      static: true
+    - class_name: "fabricPathEp"
+      target_dn: "topology/pod-1/paths-101/pathep-[eth1/2]"
+      relation_resource_name: "fibre_channel_path"
+      static: true
+
+fvRsNodeAtt:
+  resource_required:
+    - "encapsulation"
+  datasource_required:
+    - "encapsulation"
+  documentation:
+    mode: "The static association mode with the path of the %s object."
+    encap: "The VLAN encapsulation of the %s object."
+  test_values:
+    default:
+      encapsulation: "vlan-101"
+    all:
+      encapsulation: "vlan-100"
+    resource_required:
+      encapsulation: "vlan-101"
+    datasource_required:
+      encapsulation: "vlan-101"
+    datasource_non_existing:
+      target_dn: "topology/pod-2/paths-201/pathep-[eth1/1]"
+  parents:
+    - class_name: "fvAEPg"
+      parent_dependency: "fvAp"
+      parent_dn: "aci_application_epg.test.id"
+      target_classes:
+        - "fabricNode"
+  targets:
+    - class_name: "fabricNode"
+      target_dn: "topology/pod-1/node-101"
+      relation_resource_name: "static_leaf"
+      static: true
+    - class_name: "fabricNode"
+      target_dn: "topology/pod-1/node-102"
+      relation_resource_name: "static_leaf"
+      static: true
+
+fvRsDomAtt:
+  default_values:
+    untagged: "no"
+    vnetOnly: "no"
+    ipamEnabled: "no"
+  ignores:
+    - "apiMode"
+  ignore_properties_in_test:
```

Comment:
This would do away with the additional complexity in the resource_example templates like having:
 {{- else if .IgnoreInTest}}
We would only need .IgnoreInTest in testvars and the the code for ignoreTestProperty() in the generator will only return a bool value.

Reply:
The reason why I did like I did is because we use the test values for populating the examples. But since we exclude the values from testing because we do not have the correct version running we cannot do the lookup here. Let me know if you want me to spend time after my holiday to see how to do this differently but think it would require more refactor work.